web/src/app/services/ServiceAlerts.js

```diff
@@ -48,24 +57,30 @@ export default function ServiceAlerts(props) {
     return 'close'
   }
 
-  const getMenuOptions = () => {
-    return [
-      {
-        label: 'Acknowledge All Alerts',
-        onClick: handleClickAckAll,
-      },
-      {
-        label: 'Close All Alerts',
-        onClick: handleClickCloseAll,
-      },
-    ]
-  }
+  const filter = (
+    <Grid className={classes.filter} container spacing={2} alignItems='center'>
+      <Grid item>
+        <Button variant='outlined' color='primary' onClick={handleClickAckAll}>
+          Acknowledge All
+        </Button>
+      </Grid>
+      <Grid item>
+        <Button
+          variant='outlined'
+          color='secondary'
```

Comment:
There should only be 1 primary action per page which is why this is set to secondary, but since there's already a primary button for creating an alert these should actually both be secondary actions